wire/msgcftypes_test.go

```diff
@@ -0,0 +1,305 @@
+// Copyright (c) 2019 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"bytes"
+	"github.com/davecgh/go-spew/spew"
+	"io"
+	"reflect"
+	"testing"
+)
+
+// TestCFTypes tests the MsgCFTypes API
```

Comment:
Please keep non-inline comment consistent with the rest of the codebase by ending them with a period.  I'll only comment on this one, but the same applies throughout.